$OpenBSD$

debug strlcpy, include iostream for this purpose
try strlcpy with strlen + 1 (not perfect - strnlen would be better)

Index: code/qcommon/cvar.cpp
--- code/qcommon/cvar.cpp.orig
+++ code/qcommon/cvar.cpp
@@ -1395,7 +1395,7 @@ static void Cvar_Realloc(char **string, char *memPool,
 	if(string && *string)
 	{
 		char *temp = memPool + memPoolUsed;
-		strcpy(temp, *string);
+		strlcpy(temp, *string, strlen(memPool) + 1 - memPoolUsed);
 		memPoolUsed += strlen(*string) + 1;
 		Cvar_FreeString(*string);
 		*string = temp;
